### DISCUSSION TITLE
simpleomp里task队列的这部分代码是否存在泄漏呢？

### DISCUSSION THREAD

**Discussion by ZhuXinyan** [Tencent/ncnn#4397] [General]
master/src/simpleomp.cpp Line 549
好像没看到tasks在哪里被释放。

`ncnn::KMPTask* tasks = (ncnn::KMPTask*)alloca((num_threads - 1) * sizeof(ncnn::KMPTask));`

**Comment by nihui** [comment]
alloca 是栈上分配的

**Reply by ZhuXinyan** [reply]
哦哦是哦，谢谢大佬
